From 26810862b21e2b2b38e767b05e6cd17bd2ffa047 Mon Sep 17 00:00:00 2001
From: electrikjesus <electrikjesus@gmail.com>
Date: Fri, 1 Dec 2023 21:04:24 -0500
Subject: [PATCH 066/105] Add a way to override hw_timeout_multiplier

Set's the watchdog timeout multiplier (#*5s) from kernel cmdline value.

Example:
FORCE_HW_TIMEOUT_MULTIPLIER=3
---
 init.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/init.sh b/init.sh
index 2fae0db..bd1d186 100644
--- a/init.sh
+++ b/init.sh
@@ -1435,6 +1435,10 @@ for c in `cat /proc/cmdline`; do
 						# options: 0, 1
 						set_property persist.wm.debug.hide_navbar_window "$FORCE_HIDE_NAVBAR_WINDOW"
 						;;
+					FORCE_HW_TIMEOUT_MULTIPLIER=*)
+						# Force hw timeout multiplier, # X 5s
+						# options: (integer)
+						set_property ro.hw_timeout_multiplier "$FORCE_HW_TIMEOUT_MULTIPLIER"
 				esac
 				[ "$SETUPWIZARD" = "0" ] && set_property ro.setupwizard.mode DISABLED
 			fi
-- 
2.34.1

